package-manifest.toml

```diff
@@ -36,14 +36,31 @@
 
 [package.omicron-sled-agent]
 service_name = "sled-agent"
+only_for_targets.image_type = "standard"
 source.type = "local"
 source.rust.binary_names = ["sled-agent"]
 source.rust.release = true
 source.paths = [ { from = "smf/sled-agent", to = "pkg" } ]
 output.type = "tarball"
 
+# Installinator is a special package: we do not want it bundled into the normal
+# omicron install; it duplicates some of sled-agent's work and would conflict
+# with it if run side-by-side, and it's completely useless! We only want to
+# build it for inclusion in the trampoline ("recovery") image, which we indicate
+# here via `only_for_target.image_type`.
+[package.installinator]
+service_name = "installinator"
+only_for_targets.image_type = "trampoline"
+source.type = "local"
+source.rust.binary_names = ["installinator"]
+source.rust.release = true
+source.paths = [ { from = "smf/installinator", to = "pkg" } ]
+output.type = "tarball"
+output.intermediate_only = true
```

Comment:
I don't think `intermediate_only` should be set here

Reply:
Good catch; this was from an earlier experiment before adding the `image_type`.